astroquery/noao/core.py

```diff
@@ -0,0 +1,91 @@
+"""
+Provide astroquery API access to OIR Lab Astro Data Archive (natica).
+
+This does DB access through web-services. It allows query against ALL
+fields in FITS file headers.
+
+Also possible but not provided here: query through Elasticsearch DSL.  ES will
+be much faster but much more limitted in what can be used in query.  """
+
+# Python library
+# External packages
+import requests
+import astropy.table
+# Local packages
+
+# 3. local imports - use relative imports
```

Comment:
similarly, all these comments - they're leftovers from the template, and just provide instructions to you as the developer